libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -256,29 +248,29 @@ namespace SohImGui {
         }
     }
 
-    void Draw() {
-
+    void Draw1() {
```

Comment:
I think you should change this to something more recognizable like RenderMenu and RenderFramebuffer for Draw2